src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 ## Unreleased
 
-* Added `IEndpointRouteBuilder` extension methods to help with Prometheus
-  middleware configuration on ASP.NET Core
-  ([#3295](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3295))
```

Comment:
Was the removal of this changelog entry intentional?

Reply:
Good catch! I removed that entry for the listener project changelog and did a copy/paste of the changelog to this project. Fixed!